packages/appstore/src/components/wallet-content/wallet-transfer-block.tsx

```diff
@@ -11,11 +11,16 @@ type TProps = {
 };
 
 const WalletTransferBlock = observer(({ wallet_account }: TProps) => {
-    const { traders_hub, ui } = useStore();
+    const { traders_hub, ui, client } = useStore();
     const { setIsWalletModalVisible } = ui;
     const { setWalletModalActiveWalletID, setWalletModalActiveTab } = traders_hub;
+    const { accounts } = client;
 
-    const { currency, balance, loginid } = wallet_account;
+    const { linked_to } = wallet_account;
```

Comment:
@sergei-deriv where are you setting `linked_to` for `wallet_account`?

Reply:
@mahdiyeh-deriv we received it in `authorize` response and in result we have this property in `useWalletsList` hook